tycho-extras/tycho-p2-extras-plugin/src/test/java/org/eclipse/tycho/plugins/p2/extras/PublishFeaturesAndBundlesMojoTest.java

```diff
@@ -94,10 +97,12 @@ private void initLegacySupport(List<MavenProject> projects, MavenProject current
         buildContext.setSession(session);
     }
 
-    private void generateContentToBePublished(File repositoryFolder) throws IOException {
+    private void generateContentToBePublished(String repositoryFolder) throws IOException {
```

Comment:
I don't think replacing a File by a String is a good idea; maybe by a Path?

Reply:
Fixed with latest commit in the series.